File: fixtures/python/get_with_env_var.py
import os
import requests

DO_API_TOKEN = os.getenv('DO_API_TOKEN')

headers = {
    'Content-Type': 'application/json',
    'Authorization': f"Bearer {DO_API_TOKEN}",
}

params = [
    ('type', 'distribution'),
]

response = requests.get('https://api.digitalocean.com/v2/images', headers=headers, params=params)

File: test/fixtures/php-guzzle/get_with_env_var.php
<?php
require 'vendor/autoload.php';

use GuzzleHttp\Client;

$client = new Client();

$response = $client->get('http://localhost:28139/v2/images', [
    'query' => [
        'type' => 'distribution'
    ],
    'headers' => [
        'Content-Type'  => 'application/json',
        'Authorization' => 'Bearer ' . getenv('DO_API_TOKEN') ?? ''
    ]
]);
